src/utilities/bootstrapping-utilities.js

```diff
@@ -32,9 +32,10 @@ export const addAccessibility = (children, slidesToShow, currentSlide) => {
   }
 };
 
-export const getValidChildren = children => {
+export const getValidChildren = (children, rtl) => {
   // .toArray automatically removes invalid React children
-  return React.Children.toArray(children);
+  const childrens = React.Children.toArray(children);
```

Comment:
"children" is already plural, can we come up with a better name for this variable since it doesn't matter what the argument is called?